rbac/gunicorn.py

```diff
@@ -14,5 +14,5 @@
 
 cpu_resources = int(os.environ.get("POD_CPU_LIMIT", multiprocessing.cpu_count()))
 workers = cpu_resources * int(os.environ.get("GUNICORN_WORKER_MULTIPLIER", 2))
-threads = 10
+threads = int(os.environ.get("GUNICORN_THREAD_LIMIT", 10))
```

Comment:
I think we will need to add this new variable `GUNICORN_THREAD_LIMIT` to our rbac clowdapp to make it configurable 😁

Reply:
https://github.com/RedHatInsights/insights-rbac/blob/master/deploy/rbac-clowdapp.yml